src/main/java/homework7/Task3/RomanNumber.java

```diff
@@ -0,0 +1,14 @@
+package homework7.Task3;
+
+public enum RomanNumber {
+    M(1000), D(500), C(100), L(50), X(10), V(5), I(1);
+    int weight;
```

Comment:
minor: `weigth` looks like not the best option for naming. imho `value` is better